README.md

```diff
@@ -198,3 +198,9 @@ Also, *please follow the Pull Request template and feel free to discuss how the
     - Fixed overworld snow effect.
 
 There are some mechanics, moves and abilities that are missing and being developed. Check [the project's milestones](https://github.com/rh-hideout/pokeemerald-expansion/milestones) and our [issues page](https://github.com/rh-hideout/pokeemerald-expansion/issues) to see which ones.
+
+
+## Mega Evolution Overworld Sprite Credits:
```

Comment:
Can you move these to the PR description and to a new `CREDITS.md` file? That would at least get the ball rolling for #2403